CONTRIBUTING.md

```diff
@@ -0,0 +1,619 @@
+
+# Style guide
+
+You are not required to know the style guide, **but** changes may be requested on GitHub pull requests to Tiny documentation where these guidelines are not met.
+
+The Tiny documentation style guide is based on the Readability Guideline provided by Content Design London. We have chosen this guide because it’s:
+
+- open to the community
+
+- focused on readability.
+
+The information in the following subsections gives an overview of key points and lists both exceptions and additions. Please follow the guidelines on this page when they differ from the linked information.
+
+For information the Readability Guidelines project, visit [Content Design London: Readability Guidelines](https://readabilityguidelines.co.uk/).
+
+## Keep it simple
+
+Use short, simple words where possible. Use formal language.
+
+Do not use:
+
+- Long sentences.
+
+- Slang; such as *there you go*.
+
+- Jargon; such as *leverage* and *streamline*.
+
+- Ambiguous contractions; such as *there’d*, and *it’ll*.
+
+- Latin terms; such as *i.e.*, *e.g.*, *etc.*, *vs.*, and *via*.
+
+- Metaphors; such as *cherry picking* and *nutshell*.
+
+- Complex or specialist terms; such as *chrome* (the toolbar, menu
+    bar, status bar) and *upstream*.
+
+For guidelines on using specialist terms, see: [Content Design London: Readability Guidelines - Specialist terms](https://readabilityguidelines.co.uk/clear-language/specialist-terms/).
+
+For information on:
+
+- Using simple words, see: [Content Design London: Readability Guidelines - Plain English](https://readabilityguidelines.co.uk/clear-language/plain-english/).
+
+- Short, simple sentences, see: [Content Design London: Readability Guidelines - Simple sentences](https://readabilityguidelines.co.uk/clear-language/simple-sentences/).
+
+- Words to avoid, such as jargon, metaphores, and Latin terms; see: [Content Design London: Readability Guidelines - Words to avoid](https://readabilityguidelines.co.uk/clear-language/words-to-avoid/).
+
+For a short list of some commonly used Latin terms, see: [Australian Government Style Manual - Latin shortened forms](https://www.stylemanual.gov.au/format-writing-and-structure/clear-language-and-writing-style/plain-english-and-word-choice/latin-shortened-forms).
+
+## Use US English (en\_US) spelling
+
+Use United States English. For example:
+
+- "behavior" rather than "behaviour".
+
+- "canceled" rather than "cancelled".
+
+## Use proper names for programs and languages
+
+When referring to the name of a development language, use the proper name or the industry convention.
+
+For example:
+
+- "CSS" not "css"
+
+- "jQuery" not "Jquery"
+
+- "React" not "React-js"
+
+When using these terms in code elements (\`) or pre blocks (\`\`\`), use standard syntax. Such as:
+
+- `tinymce`
+
+- `<html>`
+
+- `var React = require('react');`
+
+When referring to a program, use the proper name.
+
+For example:
+
+- "Microsoft Internet Explorer" not "IE"
```

Comment:
```suggestion
- “Google Chrome” not “Chrome”
```

Given Internet Explorer has been deprecated for use in TinyMCE 6 (and is, generally and finally, no longer being widely used), a more current example programme’s proper name seems apropos.